apps/web/lib/components/Kanban.tsx

```diff
@@ -137,19 +132,6 @@ export const KanbanDroppable = ({
 	type: string;
 	content: ITeamTask[];
 }) => {
-	const [enabled, setEnabled] = useState(false);
-
-	useEffect(() => {
```

Comment:
@Anishali2 it was some sort of animations and you removed those?

Reply:
These are not important also react-beautiful-dnd does not provide these type of animation scenarios

Reply:
@Anishali2 I don't understand what you mean by "not important"? If another dev did that, it is possible it WAS important (e.g. some requirement), so what was that animation doing, and why did you remove it?

Reply:
We cannot considered that if someone do this thing he's 100% perfect might be he cannot know something or maybe some other reason 
Sir if you need that I'll add but i cannot see any useful usege of this code let me explain 
he is using gaps in drag and drop parent divs and these gaps are making animation glitch so he tried to manully fix that issue by using custom animation.
Hope you understand my point

Reply:
@Anishali2 sure, we never know, but ... "he is using gaps in drag and drop parent divs" and you don't use those gaps? Or you still use them but you managed to fix that "glitch" some other way, I want to understand?

Reply:
Ok let me explain it again 
when you use gaps in parent divs the draggable widgets react-beautiful-dnd don't know if there is gaps exist or not.
So it move the widget without gaps and when the element is dropped it gets the gaps from parent div as because the element is added in the html tree, it added the gaps so when it add it shows a little bit of glitch of moving the div from left to right.

This behaviour also exist in dnd-kit I see the code of dnd-kit and react-beautiful-dnd and dnd-kit uses most of the things from react-beautiful-dnd.The best way to solve this issue is adding padding or margin.
So why he don't added the margin because it added the space from start divs and he don't understand what to do. I'm not saying that this person is wrong but the senario i see in this code tells me this story 
So the person who is working on this thing don't figure out this thing and added the custom animation to fix that glitch but it won't help